### PR TITLE
[WIP] [Checkbox] Fix Inconsistent styling for checkbox component

### LINKED ISSUE #492: Inconsistent styling for checkbox component
# Issue summary

I don't know if this is an issue for Polaris or Chrome, but when viewing the checkbox component in Chrome on a display scaled at anything other than 100%, the checkbox gets inconsistent drop shadows. I didn't see the same behaviour in Firefox or Edge, so I'm not sure if it's a problem for you guys or an inherent Chrome problem.

## Expected behavior

Styling should be consistent regardless of scaling. At standard 100% scaling, the checkboxes look like:

image

## Actual behavior

The checkboxes have inconsistent drop shadows in Chrome at greater than 100% scaling (125% tested in this case):

image

## Steps to reproduce the problem

1. Display a form with a checkbox
2. Change display scaling to 125%

## Reduced test case

[Codesandbox](https://codesandbox.io/s/7yxl94pqkq)

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 2.12.1
- Browser: Chrome
- Device: PC
- Operating System: Windows 10

### PR DESCRIPTION
**Pull request by tanhengyeow**
### WHY are these changes introduced?

Resolves #492 

### WHAT is this pull request doing?
1. Removed `background-color` as it contributes to showing inconsistent shadows for different scaling.
2. Used `shadow(base)` to show a consistent border for different scaling.

## 

### How to 🎩

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import * as React from 'react';
import {Page, Checkbox} from '@shopify/polaris';

interface State {}

export default class Playground extends React.Component<never, State> {
  render() {
    return (
      <Page title="Playground">
        {/* Add the code you want to test here */}
        <Checkbox label="Pending" />
      </Page>
    );
  }
}

```

</details>

### CODE REVIEW

File: src/styles/shared/controls.scss [hunk 1]
```diff
@@ -16,9 +16,8 @@
 
 @mixin control-backdrop($style: base) {
   @if $style == base {
-    background-color: color('sky', 'dark');
```
[the remark this answers is not in the document]
Comment (dpersing) [reply src/styles/shared/controls.scss]: Sorry, just seeing this now for some reason. Border color would be fine @sarahill!

**Comment by AndrewMusgrave** [comment]
Hi @tanhengyeow thank you for your contribution 😃 Do you need any help pushing this along?

**Comment by tanhengyeow** [comment]
Hey everyone, thanks for the review 😄

I'm having exams now so I can only get back to this during December, hope that's fine :D

**Comment by BPScott** [comment]
Not a problem. Thank you for taking the time in the first place :)

**Comment by dpersing** [comment]
Hi @tanhengyeow! I hope your exams went well! Let us know if we can do anything else to help with this PR.

**Comment by danrosenthal** [comment]
I'm going to close this pull request for now @tanhengyeow. Feel free to reopen if/when you get the time

**Pull request closed without merging**